src/System.Text.Primitives/System/Text/Formatting/PrimitiveFormatter.cs

```diff
@@ -15,39 +15,39 @@ namespace System.Text
     public static partial class PrimitiveFormatter
     {
         #region Integers
-        public static bool TryFormat(this byte value, Span<byte> buffer, out int bytesWritten, TextFormat format, EncodingData encoding)
+        public static bool TryFormat(this byte value, Span<byte> buffer, out int bytesWritten, TextFormat format = default(TextFormat), EncodingData encoding = default(EncodingData))
```

Comment:
I suspect the case of passing an EncodingData is going to be much less frequent than formatting information. So, I'd argue to put the more commonly used parameter first.

I could be swayed with a better argument.